pytorch_vision_deeplabv3_resnet101.md

```diff
@@ -18,7 +18,10 @@ order: 1
 
 ```python
 import torch
-model = torch.hub.load('pytorch/vision:v0.10.0', 'deeplabv3_resnet101', pretrained=True)
+model = torch.hub.load('pytorch/vision:v0.10.0', 'deeplabv3_resnet50', pretrained=True)
```

Comment:
With the change of model + the change of input image, would you mind verifying that the new output is still reasonable?

Reply:
I think it should be as per the table there is little tradeoff

|    Model structure           |   Mean IOU  | Global Pixelwise Accuracy |
| ---------------------------- | ----------- | --------------------------|
| deeplabv3_resnet50           |   66.4      |   92.4                    |
| deeplabv3_resnet101          |   67.4      |   92.4                    |

Reply:
I was more thinking in terms of output of the notebook, e.g. make sure that the segmentation of the new image isn't completely off

Reply:
Yeah, I will have a run locally and post outputs of both the models

Reply:
Did you have a chance to check the segmentation output @oke-aditya ?
Otherwise no worries, I'll check myself just let me know :)

Reply:
Sorry for the delay! (I was away for the week) I will post them in few hours today.

Reply:
Mobilenetv3

![deeplab_mbnet](https://user-images.githubusercontent.com/47158509/133086492-522d0ca1-f129-4fb3-b79f-718395eb0680.png)

Resnet50

![deeplab_r50](https://user-images.githubusercontent.com/47158509/133086501-f8075fac-4815-497d-aa5a-1793a0fd1970.png)

Resnet101

![deeplab_r101](https://user-images.githubusercontent.com/47158509/133086503-39375c4c-1857-4c4a-9266-19d11a21507e.png)


I think resnet50 can be a good tradeoff

Reply:
Agreed! Thanks a lot of the outputs